### DISCUSSION TITLE
Is this UI correct?

### DISCUSSION THREAD

**Discussion by chrispage1** [awcodes/filament-curator#391] [Q&A]
Hi,

I've installed this plugin, followed the theming configuration, added content to tailwind config, updated vite config and imported CSS files into my theme.css file. I can see Vite giving the compiled CSS in dev tools.

However, the UI looks really bad, is the below screenshot correct?

![image](https://github.com/awcodes/filament-curator/assets/2487374/ca1a8c99-cb26-4827-92d6-efdba903f07c)

Worse still, once I've added an image to a block I can't then actually remove it or add any extra media?

Thanks,
Chris.

**Comment by awcodes** [comment]
Definitely something off in your theme setup.

https://github.com/awcodes/filament-curator/assets/3596800/4b899534-6f59-4582-be08-a22fc81b5df2

**Reply by chrispage1** [reply]
Thanks @awcodes - at least I can now see how it's meant to be. I didn't think it was right 😅

Appreciate the answer!
